chain/network/src/config.rs

```diff
@@ -446,7 +446,8 @@ impl NetworkConfig {
         self.routing_table_update_rate_limit
             .validate()
             .context("routing_table_update_rate_limit")?;
-        Ok(VerifiedConfig { node_id: self.node_id(), inner: self })
+        let my_node_id = self.node_id();
```

Comment:
Looks like this file is not changing meaningfully anymore; can we simply not touch it?

Reply:
Ah nice catch, reverted. thanks!